engines/python/src/main/java/ai/djl/python/engine/PyModel.java

```diff
@@ -121,6 +121,9 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
                     case "entryPoint":
                         entryPoint = value;
                         break;
+                    case "rolling_batch":
```

Comment:
In my opinion, we should not rely on `rolling_batch` property as rolling_batch will be enabled by default and it should be optional argument.

Reply:
It's optional, for TRT container, if rolling_batch is not set or "auto", we will default to trtllm.

Reply:
ignore this, I see it's handled below.